### PR TITLE
Fix edit clipping

### LINKED ISSUE #7205: Edit Button in Location Managment is Clipping out
**Describe the bug**
Edit Button in Location Managment is Clipping out

**To Reproduce**

1. Go to https://care.ohc.network/facility/0421de45-3541-4652-862d-79cec4acb6af/location
2. See error

image

### PR DESCRIPTION
**Pull request by ProCode2**
## Proposed Changes

- Fixes #7205 
- Using `word-break: break-word` to break long words if possible which does not clip the edit button anymore.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: Discard changes made to `package-lock.json`

File: src/Components/Facility/LocationManagement.tsx [hunk 1]
```diff
@@ -79,16 +79,17 @@ const Location = ({
 }: LocationModel) => (
   <div className="flex h-full w-full flex-col rounded border border-gray-300 bg-white p-6 shadow-sm transition-all duration-200 ease-in-out hover:border-primary-400">
     <div className="flex-1">
-      <div className="flex w-full items-center justify-between gap-2">
-        <div className="flex items-center gap-3">
+      <div className="flex w-full items-start justify-between gap-2">
+        <div className="flex items-start gap-3">
           <p
-            className="break-words text-xl font-medium"
+            style={{ wordBreak: "break-word" }}
```
Comment (rithviknishad) [inline src/Components/Facility/LocationManagement.tsx]: Use tailwindcss to achieve the same

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Facility/LocationManagement.tsx (hunk 1 above)
Comment (ProCode2) [reply src/Components/Facility/LocationManagement.tsx]: used the `break-all` utility from tailwind instead.

**Review by rithviknishad — APPROVED** [review]

**Review by Ashesh3 — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Pull request merged**